Task_TheGridProblem/index.css

```diff
@@ -1 +1,79 @@
-/* Start coding below for GRID PROBLEM TASK */
+.container{
+  display:grid;
+  grid-template-rows: 200px 200px 200px;
```

Comment:
It is recommended to use a unit (e.g. px) for the grid-template-rows property to ensure consistent rendering across browsers.